zproject/default_settings.py

```diff
@@ -90,9 +90,21 @@
 # Other auth
 SSO_APPEND_DOMAIN: Optional[str] = None
 
+VIDEO_ZOOM_API_URL: str = 'https://api.zoom.us/'
 VIDEO_ZOOM_CLIENT_ID = get_secret('video_zoom_client_id', development_only=True)
 VIDEO_ZOOM_CLIENT_SECRET = get_secret('video_zoom_client_secret')
 
+# Webex Video Call Settings
+VIDEO_WEBEX_API_URL: str = 'https://webexapis.com/'
+VIDEO_WEBEX_CLIENT_ID = get_secret('video_webex_client_id', development_only=True)
+VIDEO_WEBEX_CLIENT_SECRET = get_secret('video_webex_client_secret')
+# Length of alphanumeric password generated when a Webex meeting is created
+VIDEO_WEBEX_PASSWORD_LENGTH = 11
+# Length of Webex Meeting in minutes
+VIDEO_WEBEX_MEETING_LENGTH = 60
+VIDEO_WEBEX_AUTO_RECORD_MEETING = False
+VIDEO_WEBEX_ANY_USER_CAN_COHOST = False
```

Comment:
I think this is OK for an initial version, but longer-term I imagine we'll want to somehow adjust the UI to allow the user to specify these settings at meeting creation time (E.g. maybe the video call button should pop up a modal to make these selections).  Maybe worth adding a comment noting that plan.